iocage_lib/iocage.py

```diff
@@ -897,7 +897,21 @@ def exec(self,
             login_flags = self.get('login_flags').split()
             console_cmd = ['login', '-p'] + login_flags
 
-            ioc_exec.InteractiveExec(console_cmd, path, uuid=uuid)
+            try:
+                ioc_exec.InteractiveExec(console_cmd, path, uuid=uuid)
+            except BaseException as e:
+                ioc_common.logit(
+                    {
+                        'level': 'ERROR',
```

Comment:
```suggestion
                        'level': 'EXCEPTION',
```

Reply:
An exception has already been raised by `InteractiveExec`, should we raise another one in this case ?

Reply:
Yes otherwise we lose the nice exception and end up with a stacktrace